poem/Poem/frontend/react/YumRepos.js

```diff
@@ -92,7 +92,7 @@ export const YumRepoList = (props) => {
       Filter: SelectColumnFilter,
       filterList: listTags
     }
-  ]);
+  ], [isTenantSchema, listTags]);
```

Comment:
...was wondering why here simply empty array `[]` is not enough?

Reply:
It is, it would work, but linter then complains, and it bothers me :)